cmd/dep/status.go

```diff
@@ -206,7 +245,18 @@ func (out *templateOutput) BasicHeader() error { return nil }
 func (out *templateOutput) BasicFooter() error { return nil }
 
 func (out *templateOutput) BasicLine(bs *BasicStatus) error {
-	return out.tmpl.Execute(out.w, bs)
+	data := struct {
+		ProjectRoot, Constraint, Version, Revision, Latest string
+		PackageCount                                       int
+	}{
```

Comment:
Any reason not to reuse [rawStatus](https://github.com/zkry/dep/blob/4b3ad2c3a27df91cae38fdabaaa8cd43694281ba/cmd/dep/status.go#L399-L406) here?

Reply:
I thought of that too actually. It probably would make more sense to use it instead of creating an anonymous struct.